### PR TITLE
Updated tool info

### PR DESCRIPTION
**Pull request by jrbeaumont**
Update info on the concepts tool, informing users where to get it, that all concepts mentioned can be used in it, and that it is also integrated into Workcraft, and can be used with that to visualise, and automatically verify and synthesize using other built in tools.

@snowleopard: What do you think?

### CODE REVIEW

File: _posts/2015-09-10-concepts.md [hunk 1]
```diff
@@ -32,18 +31,25 @@ described using various operators which have been defined in a Haskell
...
+specifications can be found here. All concepts displayed in this series is
+written to be used with the tool, and is directly translatable to an STG.
+
 A graphical front end,
-[Workcraft](http://www.workcraft.org/),will eventually be used for
-viewing *Signal Transition Graphs* (STGs) converted from concepts using
-an algorithm, which will be described in these posts.
+[Workcraft](http://www.workcraft.org/), is used for
+viewing *Signal Transition Graphs* (STGs) translated from concepts using
+an the concepts tool, which is integrated to Workcraft, along with many other
```
Comment (snowleopard) [inline _posts/2015-09-10-concepts.md]: an the

File: _posts/2015-09-30-buffer.md [hunk 2]
```diff
@@ -12,19 +12,19 @@ In this post, we will discuss one of the most simple logic gates, a *buffer*.
 
 ![Buffer schematic]({{ site.url }}{{ site.baseurl }}/images/buffer_schematic.svg)
 
-A buffer takes an input signal, and bputs a signal in the same state as the
-input signal, the bput simply follows the input. For example, if the input
-signal changes from 0 to 1, the bput will eventually change from 0 to 1. We
-say the bput changes ‘eventually’ because the bput changes after some
+A buffer takes an input signal, and outputs a signal in the same state as the
```
Comment (snowleopard) [inline _posts/2015-09-30-buffer.md:15]: :)

**Comment by snowleopard** [comment]
Looks fine to me! Go ahead.

File: _posts/2015-09-30-buffer.md (hunk 2 above)
Comment (jrbeaumont) [reply _posts/2015-09-30-buffer.md:15]: Wait, what's this? :S

Comment (snowleopard) [reply _posts/2015-09-30-buffer.md:15]: I was just amused by the susbstitution artefact `bput` (clearly from replacing `out`s with `b`s).

Comment (jrbeaumont) [reply _posts/2015-09-30-buffer.md:15]: Oh output vs bput? 
Oh dear, this must be some weird mix of branch,
I didn't do this :S

**Pull request merged**